src/mixins/java/org/spongepowered/common/mixin/api/mcp/tileentity/CommandBlockTileEntityMixin_API.java

```diff
@@ -59,8 +59,8 @@ public DataContainer toContainer() {
         container.set(Constants.TileEntity.CommandBlock.STORED_COMMAND, this.getCommandBlockLogic().getCommand());
         container.set(Constants.TileEntity.CommandBlock.SUCCESS_COUNT, this.getCommandBlockLogic().getSuccessCount());
         container.set(Constants.TileEntity.CUSTOM_NAME, this.getCommandBlockLogic().getName());
-        container.set(Constants.TileEntity.CommandBlock.DOES_TRACK_OUTPUT, this.getCommandBlockLogic().shouldTrackOutput());
-        if (this.getCommandBlockLogic().shouldTrackOutput()) {
+        container.set(Constants.TileEntity.CommandBlock.DOES_TRACK_OUTPUT, this.getCommandBlockLogic().shouldReceiveErrors()); // bad MCP name for shouldTrackOutput
```

Comment:
Whereas you are correct (and is the case for names on both sides of the field)...don't leave comments in the code for these. Just adds noise.